Source/VMware.PSDesiredStateConfiguration/Tests/Required Dsc Resources/MyDscResource/DSCResources/CompositeResourceTest/CompositeResourceTest.psd1

```diff
@@ -0,0 +1,26 @@
+@{
```

Comment:
The License text is missing here. The same goes for all other files without License text.